perf/flame/flame.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+WD=$(dirname "${0}")
+WD=$(cd "${WD}" && pwd)
+
+FLAMEDIR="${WD}/FlameGraph"
+
+if ! which c++filt > /dev/null; then
+    echo "Install c++filt to demangle symbols"
+    exit 1
+fi 
+
+cd "${WD}" || exit -1
+
+if [[ ! -d ${FLAMEDIR} ]];then
+    git clone https://github.com/brendangregg/FlameGraph
```

Comment:
I'm tempted to recommend just printing an error message here instead of trying to automagically fix things.  the master branch may not be very stable and some people might find it a surprising side effect.